NEWS.md

```diff
@@ -1,6 +1,6 @@
 # NEWS
 
-# SOILWAT2 v7.0.0-9000
+# SOILWAT2 v7.0.0
```

Comment:
Why did commit 0bcd6b205b80455be030fea63a44fd2fb84376d0 update from branch "release/devel_v7.0.0" (which no longer existed on origin at that time) instead from latest development branch "release/devel_v7.1.0" as requested? Please update